tsdb/block_test.go

```diff
@@ -125,6 +128,45 @@ func TestCorruptedChunk(t *testing.T) {
 				testutil.Equals(t, chunks.ChunksFormatVersionSize, n)
 			},
 			errors.New("invalid chunk format version 0"),
+			nil,
+		},
+		"chunk not enough bytes to read the chunk length": {
+			func(f *os.File) {
+				// Truncate one byte after the segment header.
+				err := f.Truncate(chunks.SegmentHeaderSize + 1)
+				testutil.Ok(t, err)
+			},
+			nil,
+			errors.New("segment doesn't include enough bytes to read the chunk size data field - required:21, available:9"),
```

Comment:
Would you mind clarifying why 21 bytes are required?  

Are 13 bytes enough to read the chunk size?  
// SegmentHeaderSize (8 bytes) + binary.MaxVarintLen32 (5 bytes)

related to comment https://github.com/prometheus/prometheus/pull/6040/files#r326134782

Reply:
yes correct, updated.